EXPERIMENTS.md

```diff
@@ -0,0 +1,23 @@
+# Experiments
+
+We frequently introduce new experimental features to the agent. You can use the `--experiment` flag, or the `experiment` config value to opt-in to them and test them out.
+
+```bash
+buildkite-agent start --experiment experiment1 --experiment expertiment2
+```
```

Comment:
Could we add a config file example too? Maybe even above this? Only because I think that's the most common way people configure their agent, and people might not be sure of the array/list syntax.

Reply:
Yup, can do!